mingw-w64-python-os-win/PKGBUILD

```
@@ -0,0 +1,69 @@
+# Maintainer: Jeremy Drake <github@jdrake.com>
+
+_realname=os-win
+pkgbase=mingw-w64-python-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-python-${_realname}")
+pkgver=5.9.0
+pkgrel=1
+pkgdesc="Windows/Hyper-V library for OpenStack projects (mingw-w64)"
```

```suggestion
pkgdesc="Windows/Hyper-V python library for OpenStack projects (mingw-w64)"
```
should I specify "python" in pkgdesc?  Otherwise the descriptions were copied from the project itself somewhere.

---

Isn't it obvious that the package is a python library from the package name? Though adding it would not cause any confusion 😃